src/web/mjs/connectors/KomikTap.mjs

```diff
@@ -7,11 +7,19 @@ export default class KomikTap extends WordPressMangastream {
         super.id = 'komiktap';
         super.label = 'KomikTap';
         this.tags = [ 'manga', 'webtoon', 'hentai', 'indonesian' ];
-        this.url = 'https://92.87.6.124';
+        this.url = 'https://komiktap.in';
         this.path = '/manga/list-mode/';
+        this.hostPattern = '';
     }
 
+    async _initializeConnector() {
+        const response = await fetch(this.url);
+        const newUrl = new URL(response.url);
+        this.url = newUrl.origin;
+        this.hostPattern = newUrl.host;
+        console.log(`Assigned URL '${this.url}' to ${this.label}`);
+    }
     canHandleURI(uri) {
-        return /komiktap\.in|194\.233\.66\.232|92\.87\.6\.124/.test(uri.hostname);
+        return new RegExp(this.hostPattern).test(uri.hostname);
```

Comment:
host needs to be escaped before processing as a regexp as the dot is meaningfull :)
but ...
just use string.includes() without regexp.

Reply:
Using a string pattern, dots in ip or in host doesnt need to be escaped. 
Direct regex pattern (with /mypattern/) needs escaping.

I tested it before posting, but ill switch to includes. 
FYI, dual regexp 'komiktap\.in|194\.233\.66\.232' was recognised by clipboard but nothing worked with komiktap\.in urls in entered. Better just using only one host.

Reply:
Hum, _initializeConnector isnt called before canHandleURI, so the host is not updated.
gotta have to do this

```
    canHandleURI(uri) {
        if (!this.initialized) super._initialize();
        return uri.href.includes(this.url);
    }
```

Reply:
Used includes, and had to used async cause stuff is not always initializated before. Its better than calling initializate() and it works.

Reply:
you cannot switch to async "just like that" :)
the connector code is using it as a synchronous call
https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/system/ClipboardConnector.mjs#L51

Reply:
You think it would be better to call initialize? fine then

Reply:
lint error ;)

Reply:
https://smallpetselect.com/wp-content/uploads/2016/06/bunniez.hubpages.com_.png